internal/pkg/inject/consts_test.go

```diff
@@ -6,7 +6,7 @@ package inject
 import (
 	"testing"
 
-	"github.com/hashicorp/go-version"
+	"github.com/Masterminds/semver/v3"
```

Comment:
There is also https://pkg.go.dev/golang.org/x/mod/semver

But I have not checked if it would be better. Possibly it is lighter.

Reply:
I guess you mean https://github.com/open-telemetry/opentelemetry-go-instrumentation/issues/1753. Resolving